include/deal.II/numerics/vector_tools_hermite.templates.h

```diff
@@ -0,0 +1,597 @@
+// ---------------------------------------------------------------------
```

Comment:
The comments in include/deal.II/numerics/vector_tools_hermite.templates.h seem to be not addressed, yet.